### PR TITLE
release notes for v1.0.2

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## v1.0.2
+* Fix issue #13533 - only use systemd when present
```
Comment (Luap99) [inline RELEASE_NOTES.md:4]: This should link to the podman issue, #13533 does not exists in this repo

**Review by lsm5 — APPROVED** [review]
LGTM after @Luap99 's concern addressed.

**Pull request merged**